sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java

```diff
@@ -1013,7 +1013,7 @@ public Accumulator decode(InputStream inStream) throws CoderException, IOExcepti
           public Accumulator decode(InputStream inStream, Coder.Context context)
               throws CoderException, IOException {
             return new Accumulator(
-                StringUtf8Coder.of().decode(inStream, context),
+                StringUtf8Coder.of().decode(inStream, context.nested()),
```

Comment:
May be useful to extract these into separate statements to avoid ambiguity about parameter evaluation order.